contents/blog/how-to-start-a-growth-team.md

```diff
@@ -0,0 +1,123 @@
+---
+date: 2023-07-26
+title: "How to start a growth team (as an engineer)"
```

Comment:
I gave some thought to a new title, but I still really like this one. It's quite eye catching IMO